source/manual/2nd-line-drills.html.md

```diff
@@ -92,3 +93,48 @@ Either before merging the change, or part way through the continuous deployment
 Follow the deployment pipeline in Jenkins. Confirm that no further environment deployments are triggered. For example, if you implemented the deploy freeze just after the app was deployed to Staging, confirm that the app was then not automatically deployed to Production.
 
 Remove the code freeze, then manually push the changes to all remaining environments so that they're in sync.
+
+## Drill getting a password out of govuk-secrets
+
+Most of our shared accounts are stored in this repo. Follow the steps to [get a password](https://github.com/alphagov/govuk-secrets/tree/main/pass#get-a-password), a useful one for 2nd line is the [Technical 2nd line Dashboard](https://alphagov.github.io/frame-splits/index.html?title=2nd+Line+Dashboard&layout=2x1-responsive&url%5B%5D=https%3A%2F%2Fgovuk-secondline-blinken.herokuapp.com%2Fblinken.html&url%5B%5D=https%3A%2F%2Fgrafana.production.govuk.digital%2Fdashboard%2Ffile%2F2ndline_health.json&url%5B%5D=https%3A%2F%2Fgovuk-zendesk-display-screen.herokuapp.com&url%5B%5D=) which shows a high level overview of the state of GOV.UK environments.
+Upon loading this dashboard it will ask for the `govuk-zendesk-display-screen` credentials.
+
+## Drill assigning a user to their publisher in data.gov.uk
+
+[Log into our shared data.gov.uk publisher account](/manual/data-gov-uk-2nd-line.html#logging-into-the-publisher). Pick a [publisher](https://ckan.publishing.service.gov.uk/organization) to do a hypothetical walk though of [Assign users to publishers](/manual/data-gov-uk-2nd-line.html#assign-users-to-publishers-setting-user-permissions).
+
+## Drill redirecting a route
+
+Follow the [Removing a route created in the Short URL Manager](/manual/redirect-routes.html#removing-a-route-created-in-the-short-url-manager) and [Removing a route completely so it can be replaced with another route](/manual/redirect-routes.html#removing-a-route-completely-so-it-can-be-replaced-with-another-route) instructions.
+Do this on Integration or Staging.
+
+## Drill changing a slug and create a redirect
```

Comment:
Similar to previous comment - these tasks feel a bit too granular, meaning 1) the devs drilling the task don't gain an awful lot from it, and 2) it takes longer for all drills to be processed. Would it make sense to combine the above two into a "Drill creating and changing redirects" 'super task'?